com.inochi2d.inochi-session.yml

```diff
@@ -0,0 +1,67 @@
+id: com.inochi2d.inochi-session
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.llvm14
+command: inochi-session
+finish-args:
+  - --device=all          # OpenGL rendering, webcams (may be required? not sure)
```

Comment:
if you are not sure, either make sure, or restrict to `dri`.

Reply:
Nice catch! I forgot to change this :sweat_smile:. this app will require to connect to webcams in the future, but that's not the case for now, so I changed it to `dri.`